book/modules/ROOT/pages/introduction.adoc

```diff
@@ -7,27 +7,39 @@
 
 Welcome to The Examples Book! This book contains a collection of information and examples that can be used to reinforce topics learned in https://datamine.purdue.edu[The Data Mine] seminar at https://purdue.edu[Purdue University]. The Examples Book is open to anyone! Even if you aren't a student at Purdue we hope you find helpful content here. 
 
-This book is broken down into "core" topics. These topics are listed in the options below. They range from getting started with data science to information for the corporate partners who want to sponsor a project with us. 
-
-In addition, the live version of the "core" book contains the most up-to-date slate of projects students in The Data Mine are working on. All projects from previous semesters live in the xref:projects:ROOT:introduction.adoc[Projects Archive] appendix and are open to the public.
-
 [TIP]
 ====
-If you're here to explore or learn more about data science we recommend starting with the `Data Science` button below. Have fun!
+Where to go for specific topics:
+
+* Question about predictive models, data analysis, or visualization? Check out `Data Science`.
+* Need to know more about containers or research computing? Check out `Data Engineering`. 
+* Learning R or Python. `Coding Languages` might be helpful. 
+* Want more info on The Data Mine, corporate partners projects, or student course resources? Check out `Corporate Partners`.
+* Looking to see the projects that we use to build skills in analytics? Read through the `Course Links`.
+* Have a different questions or want to add content to The Examples Book? `Helpful Links` is the place for you. 
```

Comment:
Maybe:

Did you find a mistake in the book? Would you like to add your own content? Do you have a frequently asked question (FAQ)? `Helpful Links` is the place for you.